lexers/s/sourceslist_test.go

```diff
@@ -0,0 +1,43 @@
+package s_test
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/alecthomas/assert"
+	"github.com/alecthomas/chroma"
+	"github.com/alecthomas/chroma/lexers/s"
+)
+
+func TestSourcesList_AnalyseText(t *testing.T) {
+	tests := map[string]struct {
+		Filepath string
+		Expected float32
+	}{
+		"standard": {
+			Filepath: "testdata/sources.list",
+			Expected: 1.0,
+		},
+		"indented": {
+			Filepath: "testdata/sources-indented.list",
+			Expected: 1.0,
+		},
+		"invalid": {
```

Comment:
Do we need to test invalid cases? There's no other tests I've seen we implementing for failing cases.

Reply:
Here I wanted to make sure it works, as I translated a for loop through the lines to a single regex: https://github.com/pygments/pygments/blob/debda34e2d4f28d6d369cdafdcba4791702f63fc/pygments/lexers/installers.py#L260